source/index.html.erb

```diff
@@ -113,22 +113,23 @@ Join Us
 
 		<div class="panel-body" id="events">
 
-			<h4>2016 CloudStack Collaboration Conferences</h4>
+			<h4>2017 CloudStack Collaboration Conferences</h4>
 
 			<p>
-			In 2016, two Apache <a href="http://cloudstackcollab.org/">CloudStack Collaboration Conferences</a> events will be held in the Americas. Each event will feature morning plenary sessions, afternoon breakout sessions in user and developer tracks and a hackathon. Learn more at: 
+			In 2017, two Apache <a href="http://cloudstackcollab.org/">CloudStack Collaboration Conference</a> events will be held in the Americas. Each event will feature a lot of opportunities to learn, collaborate and engage with the CloudStack community.  The CloudStack Collaboration Conferences are valuable for both developers and users, and are a great way to get involved with the community.
```

Comment:
No, not really. This section says 'In 2017, two Apache CCC events ....' and points to a web page where only one of the two events is mentioned, but also mentions a previous (Canada) event.

Personally it is likely that I would click on the first link and expect to find the ApacheCon there, even if only a link to the proper ApacheCon site.

It is really more about what content exists on cloudstackcollab.org, so I'm fine with merging this one and discussing cloudstackcollab elsewhere.